notifications-demo/app/src/main/java/com/jwplayer/demo/notificationsdemo/DownloadImageTask.java

```diff
@@ -0,0 +1,52 @@
+package com.jwplayer.demo.notificationsdemo;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Downloads images and returns them as bitmap.
+ *
+ * Clients need to extend it and implement their own onPostExecute method.
+ */
+public abstract class DownloadImageTask extends AsyncTask<String, Void, Bitmap> {
+
+	@Override
+	protected Bitmap doInBackground(String... urls) {
```

Comment:
It seems like we are only ever downloading the first url, and returning that is a bitmap. Is the correct?